src/kOS/Module/kOSVesselModule.cs

```diff
@@ -19,6 +19,7 @@ public class kOSVesselModule : VesselModule
         private bool initialized = false;
         private Vessel parentVessel;
         private bool hasRemoteTech = false;
+        private int counterRemoteTechRefresh = 0;
```

Comment:
Can we begin this at 24 instead of at zero?  That way it only waits 1 update before trying to steer for the very first time.  Right now, I've had problems where the fact that steering is disabled for 1 second after a staging event during liftoff causes the aero effects to start deflecting the vessel from prograde in ways that feel a bit dangerously close to the flipover point.